Library/Formula/connect.rb

```diff
@@ -0,0 +1,17 @@
+class Connect < Formula
+  desc "provides SOCKS and HTTPS proxy support to SSH."
+  homepage "https://bitbucket.org/gotoh/connect/wiki/Home"
+  url "https://bitbucket.org/gotoh/connect/get/1.104.tar.gz"
+  sha256 "6ca5291ea8e4a402d875a5507ac78d635373584fd1912f2838b93e03b99730c8"
+
+  def install
+    # gcc will produce warnings but the application will work as expected
+    system ENV.cc, "connect.c", "-o", "connect", "-lresolv"
```

Comment:
We generally prefer `make` if one exists just because this is the sort of thing upstream's _should_ support rather than expecting users to be familiar enough with compiler syntax to get something to build. If there's no `make` there's not much we can do, aside ask upstream nicely if they're willing to consider adding one.